target/linux/ath79/generic/base-files/etc/board.d/03_gpio_switches

```diff
@@ -73,6 +73,10 @@ tplink,wbs510-v1|\
 tplink,wbs510-v2)
 	ucidef_add_gpio_switch "poe_passthrough" "PoE Passthrough" "20"
 	;;
+ubnt,aircube-isp|\
+ubnt,aircube-ac)
```

Comment:
ac before isp

Reply:
thanks,
as we keep changes for acb-isp -> aircube-isp change separately - order would be correct for now.
ubnt,acb-isp comes first and then ubnt,aircube-ac

i will swap them around making aircube-ac first with mr that renames acb-isp -> aircube-isp.